src/reconcile/rec_visibility.c

```diff
@@ -833,15 +840,16 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
     /*
      * Paranoia: check that we didn't choose an update that has since been rolled back.
      */
+    // WT-9417 IGNORE
     WT_ASSERT(session, upd_select->upd == NULL || upd_select->upd->txnid != WT_TXN_ABORTED);
```

Comment:
This should be a strong assertion.

Reply:
Updated in [ad25619](https://github.com/wiredtiger/wiredtiger/pull/8004/commits/ad256191cfabfcf3144de804615d722c2509cb24)